examples/basic_bq/main.tf

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+   Provider configuration
+  *****************************************/
+provider "google" {
+  version = "~> 1.20.0"
+
+  # credentials = "${file(local.credentials_file_path)}"
```

Comment:
Style: let's drop this comment since it isn't providing any value in the code.
```suggestion

```

Reply:
Will be dropped in the next release